packages/core-components/src/components/ContentElement.vue

```diff
@@ -25,57 +25,71 @@
       />
     </div>
     <ActiveUsers :size="20" :users="activeUsers" class="active-users" />
-    <template v-if="!!manifest">
-      <QuestionElement
-        v-if="isQuestion"
-        :icon="manifest.ui.icon"
-        :type="manifest.name"
-        v-bind="{
-          ...$attrs,
-          componentName,
-          embedElementConfig,
-          element,
-          references,
-          isFocused,
-          isDragged,
-          isDisabled,
-          isReadonly: props.isDisabled,
-          dense,
-        }"
-        @add="emit('add', $event)"
-        @delete="emit('delete')"
-        @focus="onSelect"
-        @link="onLink"
-        @save="onSave"
-      />
-      <component
-        :is="componentName"
-        v-else
-        v-bind="{
-          ...$attrs,
-          embedElementConfig,
-          element,
-          references,
-          isFocused,
-          isDragged,
-          isDisabled,
-          isReadonly: props.isDisabled,
-          dense,
-        }"
-        :id="`element_${id}`"
-        @add="emit('add', $event)"
-        @delete="emit('delete')"
-        @focus="onSelect"
-        @link="onLink"
-        @save="onSave"
-      />
-    </template>
-    <VSheet v-else class="py-10" color="primary-lighten-5">
-      <div class="text-h6">
-        {{ element.type.replace('_', ' ') }}
+    <VSheet
```

Comment:
Getting a bit crowded, should we consider extracting some of these into subcomponents ?